docs/reference/mount-write-mode-limitations.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Mount Write Mode Limitations
+description: Limitations of the mount write mode
+grand_parent: Features
+parent: Mount
+---
+
+# Mount Write Mode Limitations
+
```

Comment:
Let's try to make it more structured because there are many things to unpack here. 

1. Overview 
2. [Architecture and concurrency model](https://github.com/treeverse/dev/blob/main/designs/open/everest-mount/mount-write.md#architecture-and-concurrency-model)  - few bullets not long are enough but that section 
is great!
3. Committing Changes
- how it works, consistency, merge strategy, what happens after commit. 
- writes during commit may be lost, dont do it
5. Limitations 
- **sub section: functional bullets:** (NFS, no eviction for write dir its not cache, durability: restarting the mount may lead to unpredicted behavior) 
- **sub section: FS Limitations** (as mentioned below)